src/main/java/ch/njol/skript/entity/CatData.java

```diff
@@ -73,7 +80,8 @@ public EntityData getSuperType() {
 	
 	@Override
 	protected int hashCode_i() {
-		return race != null ? race.hashCode() : 0;
+		//noinspection RedundantCast - cast to prevent IncompatibleClassChangeError due to Enum->Interface change
+		return race != null ? ((Object) race).hashCode() : 0;
```

Comment:
I think it might be nicer to use something like the Objects hashcode method rather than a cast here to prevent any weird problems.

Reply:
Sure, I'll see what I can do